src/utils/prettyNum.tests.ts

```diff
@@ -0,0 +1,14 @@
+import o from 'ospec';
+import { prettyNum } from './prettyNum';
+
+o.spec('regEscape()', () => {
+	o('Escapes strings', () => {
+		o(prettyNum()).equals('0');
+		o(prettyNum(42)).equals('42');
+		o(prettyNum(123456789)).equals('123.456.789');
+		o(prettyNum(123456.789, 2, '.', ',')).equals('123.456,79');
```

Comment:
Vantar að skjala hvað gerist þegar input gildið er með færri decimals en arg2 tilgreinir..

```suggestion
		o(prettyNum(123456.789, 2, '.', ',')).equals('123.456,79');
		// þetta? (max-decumals)
		o(prettyNum(123456.7, 2, '.', ',')).equals('123.456,7');
		o(prettyNum(123456, 2, '.', ',')).equals('123.456');
		// eða þetta? (fixed decimals)
		o(prettyNum(123456.7, 2, '.', ',')).equals('123.456,70');
		o(prettyNum(123456, 2, '.', ',')).equals('123.456,00');
```

Ég held að max-decimals sé almennt betra default.
En fixed-decimals sé nauðsynleg optional hegðun (sjá comment við `prettyNum.ts` að neðan)

Reply:
Búinn að innleiða fixed decimals